doc/api/console.markdown

```diff
@@ -48,6 +48,10 @@ is used on each argument.  See [util.format()][] for more information.
 
 Same as `console.log`.
 
+### console.debug([data][, ...])
```

Comment:
Yes, I think changing the signatures would be preferred. Can you do so in a second commit?